src/_guides/language/effective-dart/usage.md

```diff
@@ -376,8 +376,9 @@ numbers.removeAt(1); // Now it only contains integers.
 var ints = List<int>.from(numbers);
 {% endprettify %}
 
-But if your goal is just to copy the iterable and preserve its original type, or
-you don't care about the type, then use `toList()`.
+If your goal is to _copy_ the iterable and preserve its original element type,
```

Comment:
`List.of()` doesn't preserve the original element type. It uses the static type, which may be different. Sometimes you want to preserve the runtime type and sometimes you want to *not* preserve the runtime type and use the static type.

I think we have basically two options:

* Provide minimal guidance (basically avoid `List.from()` with an uninferred type) and let people sort it out.

* Provide detailed guidance for all of the places where each option is best.